exercises/hamming/canonical-data.json

```diff
@@ -30,97 +30,25 @@
       "expected": 0
     },
     {
-      "description": "long identical strands",
-      "property": "distance",
-      "input": {
-        "strand1": "GGACTGA",
-        "strand2": "GGACTGA"
-      },
-      "expected": 0
-    },
-    {
-      "description": "complete distance in single nucleotide strands",
-      "property": "distance",
-      "input": {
-        "strand1": "A",
-        "strand2": "G"
-      },
-      "expected": 1
-    },
-    {
-      "description": "complete distance in small strands",
-      "property": "distance",
-      "input": {
-        "strand1": "AG",
-        "strand2": "CT"
-      },
-      "expected": 2
-    },
-    {
-      "description": "small distance in small strands",
-      "property": "distance",
-      "input": {
-        "strand1": "AT",
-        "strand2": "CT"
-      },
-      "expected": 1
-    },
-    {
-      "description": "small distance",
+      "description": "single letter different strands",
       "property": "distance",
       "input": {
-        "strand1": "GGACG",
-        "strand2": "GGTCG"
+        "strand1": "G",
+        "strand2": "T"
       },
       "expected": 1
     },
     {
-      "description": "small distance in long strands",
-      "property": "distance",
-      "input": {
-        "strand1": "ACCAGGG",
-        "strand2": "ACTATGG"
-      },
-      "expected": 2
-    },
-    {
-      "description": "non-unique character in first strand",
```

Comment:
I have never understood what kinds of implementations this case was meant to detect, and https://github.com/exercism/problem-specifications/pull/51 which added it did not provide the rationale.

However, I suppose in its original form (`AGA` vs `AGG`), it would find implementations that would checked whether the *set* of letters is the same (for example, here, declare that the two strings have distance 0 because both of them contain at least one A and at least one G).

If that is so, https://github.com/exercism/problem-specifications/pull/953 lost that ability. Since nobody has come forth in the year that it's been gone and requested that it be changed back, that's a fine indication that that was of no concern to anyone, and thus should be removed.